### PR TITLE
Added messaging python

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

File: messaging/methods/media/getMedia.md [hunk 1]
```diff
@@ -47,7 +47,10 @@ var a = b;
 {% sample lang="python" %}
 
 ```python
-# Python Example
+downloaded_media = messaging_client.get_media(MESSAGING_ACCOUNT_ID, media_id)
+f = open("file_to_write", "wb")
+f.write(downloaded_media.body)
```
Comment (ghost) [inline messaging/methods/media/getMedia.md:52]: 👍

File: messaging/methods/media/listMedia.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ var a = b;
 {% sample lang="python" %}
 
 ```python
-# Python Example
+media = messaging_client.list_media("12345")
```
Comment (ghost) [inline messaging/methods/media/listMedia.md]: Can you add example on iterating over this object? 

Also noticed we bounce between `MESSAGING_ACCOUNT_ID` and `12345`. I personally like the `MESSAGING_ACCOUNT_ID` var over the string of numbers.

File: messaging/methods/media/listMedia.md [hunk 3]
```diff
@@ -140,6 +140,6 @@ var a = b;
 {% sample lang="python" %}
 
 ```python
-# Python Example
+media = messaging_client.list_media("12345", continuation_token="token")
```
Comment (ghost) [inline messaging/methods/media/listMedia.md]: Same as above, but we'll grab with the DX-799 work to prove out how to get the next pages

File: messaging/methods/messages/createMessage.md [hunk 4]
```diff
@@ -482,7 +509,15 @@ var a = b;
...
 
 ```python
-# Python Example
+body = MessageRequest()
+body.application_id = "93de2206-9669-4e07-948d-329f4b722ee2"
+body.to = ["+12345678902", "+12345678903"]
+body.mfrom = "+12345678901"
+body.text = "Hey, check this out!"
+try:
+    messaging_client.create_message("12345", body)
```
Comment (ghost) [inline messaging/methods/messages/createMessage.md]: Can you add example of working with the response object in these examples

**Review by ghost — CHANGES_REQUESTED** [review]
I think once we're done implementing all these we should just find/replace throughout to consistently have `MESSAGING_ACCOUNT_ID` in the function call.  And be sure that @jchavez-bw picks that up for C# as well

File: messaging/methods/messages/createMessage.md (hunk 4 above)
Comment (jmulford-bw) [reply messaging/methods/messages/createMessage.md]: I put the example in 1 of them. I think I just forgot to copy-paste it to the other ones. Updated 👍

File: messaging/methods/media/listMedia.md (hunk 2 above)
Comment (jmulford-bw) [reply messaging/methods/media/listMedia.md]: Hmm I can, I'll have to do some digging around to get that.

Also will update to use the global var

**Pull request merged**